### PR TITLE
feat(series): separate editor for adding series items

### PR DESCRIPTION
**Pull request by akashgp09**
**Summary:**
Created separate editor for adding/editing/removing series item.

### CODE REVIEW

File: src/client/entity-editor/series-section/series-editor.tsx
Comment (MonkeyDo) [inline src/client/entity-editor/series-section/series-editor.tsx:143]: The addition of the `form-control-static` CSS class to this Col element or its child will align it vertically to match the `form-control` class of the input in the previous column.

To compare the margins with and without the class (with added red border to see it better):
image

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
This is looking very nice, well done :)

I think it will make the editing flow much smoother, and we are likely to adapt the same strategy for other entity types in the future (for example Works part of an Edition, with the page attribute on the left-hand side).
This is a great way to refine that workflow with a good use-case, and get feedback from the community.

As I mentioned in one of the comments below I suspect the manual reordering with drag n' drop will be coming in a following PR?

File: src/client/entity-editor/series-section/series-editor.tsx
Comment (akashgp09) [reply src/client/entity-editor/series-section/series-editor.tsx:132]: > I think it was a great way to do it, and should be added back (is that the work you mentioned that you have locally and waiting for this PR to be merged?)

Yes indeed, I will open a different PR for it once this gets merged, and also it will be easier for you to review smaller PRs.

File: src/server/routes/entity/entity.tsx
Comment (akashgp09) [reply src/server/routes/entity/entity.tsx:1196]: Cool, applying the changes in a bit :)

**Review by MonkeyDo — APPROVED** [review]
Good to go ! 🚀

**Pull request merged**
